qutebrowser/mainwindow/statusbar/command.py

```diff
@@ -259,3 +264,9 @@ def _incremental_search(self, text):
 
         if self.prefix() in ['/', '?']:
             self.got_cmd[str].emit(search_prefixes[text[0]] + text[1:])
+
+    @pyqtSlot(str)
+    def _exit_prefix(self, text):
+        if config.val.completion.exit_prefix and (not text or text[0] not in self.prefixes):
+            modeman.leave(self._win_id, usertypes.KeyMode.command,
+                          'prefix deleted', maybe=True)
```

Comment:
Why `maybe=True`? If this gets called, we're guaranteed to be in command mode, right?

Reply:
Without this it crashes after executing a command.